.github/workflows/pull-request.yaml

```diff
@@ -289,8 +283,7 @@ jobs:
       - name: Run integration tests
         run: |
           cd tracetesting
-          TRACETEST_CLI_MAIN="/tmp/tracetest-main/cli/dist/tracetest" \
-          TRACETEST_CLI_TARGET="../cli/tracetest" \
+          TRACETEST_CLI="../cli/tracetest" \
```

Comment:
I'm changing our dogfood process to use the PR CLI instead of the main CLI. By doing it we will be able to dogfood the CLI too.

Reply:
This might break things. What happens if we change something in the server/CLI which is not retro-compatible? The PR CLI will not be able to communicate with the integration instance (which is deployed using the latest main code)

Reply:
Makes sense! I'll revert the changes and keep our dogfooding using only the stable/main CLI for now. Later we can add a step just to test the CLI better.